compute/etc/sql_exporter/getpage_stuck_requests_total.libsonnet

```
@@ -0,0 +1,9 @@
+{
+  metric_name: 'getpage_stuck_requests_total',
```

To do not confuse with any pageserver-side metrics

```suggestion
  metric_name: 'compute_getpage_stuck_requests_total',
```

---

We have a bunch of other getpage_ metrics exposed, renaming this one to compute_getpage_ would make it inconsistent